documentation/api/admin/v1/archive.markdown

```diff
@@ -12,27 +12,27 @@ The `/archive` endpoint can be used for importing and exporting PuppetDB archive
 
 This endpoint can be used for streaming a PuppetDB archive into PuppetDB.
 
-### Request Format
+### Request format
 
 The request should be a multipart POST and have `Content-Type: multipart/mixed`.
 
-### URL Parameters
+### URL parameters
 
-* `archive`: Required. The archive file to import to the PuppetDB.
-* `command_versions`: Required. A JSON object mapping PuppetDB command-names to their version. The mapping for a given PuppetDB archive can be found in the archive:
-~~~shell
-        tar -xOf <my-pdb-archive>.tar.gz puppetdb-bak/export-metadata.json
+* `archive`: required. The archive file to import to the PuppetDB.
+* `command_versions`: required. A JSON object mapping PuppetDB command-names to their version. The mapping for a given PuppetDB archive can be found in the archive:
```

Comment:
preexisting, but we probably shouldn't hyphenate command-names since it's not a special thing